packages/protocol/contracts/uniswap/test/libraries/Math.sol

```diff
@@ -0,0 +1,23 @@
+pragma solidity ^0.5.13;
+
+// a library for performing various math operations
+
+library Math {
+  function min(uint256 x, uint256 y) internal pure returns (uint256 z) {
+    z = x < y ? x : y;
+  }
+
+  // babylonian method (https://en.wikipedia.org/wiki/Methods_of_computing_square_roots#Babylonian_method)
+  function sqrt(uint256 y) internal pure returns (uint256 z) {
+    if (y > 3) {
```

Comment:
We should be using SafeMath here

Reply:
This is how it actually is implemented in Uniswap, so I'd just leave like that